wfsim/load_resource.py

```diff
@@ -114,26 +114,25 @@ def __init__(self, config=None):
                     log.warning(f'{k} did not download, trying {raw_url}')
                     files[k] = raw_url
             log.debug(f'Downloaded {k} successfully')
+                             
+        self.photon_area_distribution = straxen.get_resource(files['photon_area_distribution'], fmt='csv')
 
         if config['detector'] == 'XENON1T':
             self.s1_pattern_map = make_map(files['s1_pattern_map'], fmt='json.gz')
             self.s1_light_yield_map = make_map(files['s1_light_yield_map'], fmt='json')
             self.s2_light_yield_map = make_map(files['s2_light_yield_map'], fmt='json')
             self.s2_pattern_map = make_map(files['s2_pattern_map'], fmt='json.gz')
-
-            # Garfield luminescence timing samples
-            if config['s2_luminescence_model'] == 'garfield':
-                raise NotImplementedError
+            self.fdc_3d = make_map(files['fdc_3d'], fmt='json.gz')
 
             # Gas gap warping map
-            if config['enable_gas_gap_warping']:
+            if config.get('enable_gas_gap_warping',False):
                 self.gas_gap_length = make_map(["constant dummy", 0.25, [254,]])
 
             # Photon After Pulses
-            if config['enable_pmt_afterpulses']:
+            if config.get('enable_pmt_afterpulses',False):
                 self.uniform_to_pmt_ap = straxen.get_resource(files['photon_ap_cdfs'], fmt='pkl.gz')
 
-        if config['detector'] == 'XENONnT':
+        if config.get('detector','XENONnT') == 'XENONnT' and not config.get('neutron_veto', False):
```

Comment:
@mzks Why is this change necessary?

Reply:
nVeto side doesn't use the following files, thus we can skip them for nVeto.
If we don't add them, it works.

Reply:
Ok, for a moment I was happily thinking that WFSim runs simultaneously for TPC and nVeto, while every time the `neutron_veto` config changes, it actually means we updated our context and started a new WFSim run :)